scripts/alert_emails/firestore.ts

```diff
@@ -19,6 +19,7 @@ export async function fetchAllAlertSubscriptions(
       .stream()
       .on('data', subscription => {
         const data = subscription.data();
+        data.email = subscription.id;
```

Comment:
Currently, `subscription.data()` would return the `subscribedAt` and `locations` fields, but not the `email` field. To get the email field, it seems like to we need to explicitly call the subscription's `id` field.

Reply:
Oh, nice catch. THanks for explaining!